app/tests/sticky-keys/12-same-position-mods/keycode_events.snapshot

```diff
@@ -0,0 +1,6 @@
+pressed: usage_page 0x07 keycode 0xE0 implicit_mods 0x00 explicit_mods 0x00
+pressed: usage_page 0x07 keycode 0xE1 implicit_mods 0x00 explicit_mods 0x00
+pressed: usage_page 0x07 keycode 0x04 implicit_mods 0x00 explicit_mods 0x00
+released: usage_page 0x07 keycode 0x04 implicit_mods 0x00 explicit_mods 0x00
+released: usage_page 0x07 keycode 0xE0 implicit_mods 0x00 explicit_mods 0x00
+released: usage_page 0x07 keycode 0xE1 implicit_mods 0x00 explicit_mods 0x00
```

Comment:
This seems a bit inconsistent with how sticky mods across *different* positions get stacked. There we have `ignore-modifiers` to determine whether or not they combine (with the default being not). Would it make sense to make same-position sticky keys follow the same logic?

Reply:
You are right, I should have checked. Somehow I thought it defaulted the other way around.